.changeset/playground-auth-me-503-retry.md

```
@@ -0,0 +1,14 @@
+---
+'@internal/playground': patch
+---
+
+fix(playground): retry /api/auth/me on transient 503 from auth middleware
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)